<launch>
    <!-- remember to hard code the serial of the two cameras into the pgrff_node_software -->
    <node pkg="cameras_tryphon" type="pgrff_tryphon_software2" name="cameras_node" output="screen">
      <param name="framerate" value="10" />
      <param name="camera1_info_url" value="file:///home/tryphon/ROSTryphon_groundstation/src/cameras_tryphon/launch/calibration/camera1.yaml" />
      <param name="camera2_info_url" value="file:///home/tryphon/ROSTryphon_groundstation/src/cameras_tryphon/launch/calibration/camera2.yaml" />
    </node>
</launch>
